src/main/resources/synthetic.xml

```diff
@@ -40,21 +53,70 @@
-        <property name="subjectDelimiter" hidden="true" default="|"/>
+    </type>
+    <type type="wp.PortletCloneLocaleDataSpec" extends="udm.BaseEmbeddedDeployable" description="Portlet clone locale spec">
+        <property name="locale" description="locale of the name, description and keywords"/>
+        <property name="title" description="Portlet title in specified locale" required="true"/>
+        <property name="description" description="Portlet description in specified locale" required="false"/>
+        <property name="keywords" description="Portlet keywords in specified locale" required="false"/>
     </type>
 
+    <!-- Deployeds -->
     <type type="wp.WarModule" extends="was.WarModule" deployable-type="wp.War" container-type="was.WasAppContainer" description="WAR with values configured for a deployment.">
         <property name="portalAppName" label="Portal Application Name" description="Name for this portal application" category="Portal"/>
-        <property name="portalAppUid" label="Portal Application Identifier" description="Unique identifier for this portal application as defined in the portal.xml" category="Portal"/>
+        <property name="portalAppUid" label="Portal Application Identifier" description="Unique identifier for this portal application as defined in the portlet.xml" category="Portal"/>
         <property name="portalAppConcreteUid" required="false" label="Concrete Portal Application Identifier"
-                  description="Required when deploying a IBM-API type portlet. Concrete Unique identifier for this portal application as defined in the portal.xml" category="Portal"/>
+                  description="Required when deploying a IBM-API type portlet. Concrete Unique identifier for this portal application as defined in the portlet.xml" category="Portal"/>
         <property name="portlets" kind="set_of_ci" required="true" as-containment="true" referenced-type="wp.Portlet" label="Portlets" category="Portal" description="List of portlets to register."/>
 
         <property name="modifyScript" hidden="true" required="false" default="was/portal/application/update-application.py"/>
         <property name="warInstallLocationPrefix" hidden="true" required="false" default="file://localhost/"/>
     </type>
 
+    <type type="wp.Portlet" extends="udm.BaseEmbeddedDeployed" deployable-type="wp.PortletSpec" container-type="wp.WarModule" description="Deployed portlet to WebSphere Portal runtime.">
+        <property name="portletName" description="Name of the portlet (portlet-name) as described in the portlet.xml"/>
+        <property name="uniqueName" description="Unique name for portlet" required="false"/>
+        <property name="preferences" description="Portlet preferences." required="false" kind="map_string_string"/>
+        <property name="clones" kind="set_of_ci" required="true" as-containment="true" referenced-type="wp.PortletClone" label="Clones" description="List of portlet clones to register."/>
```

Comment:
This property should be made optional, not required.  Otherwise existing deployments that don't use clones will fail